8/debian-10/rootfs/opt/bitnami/scripts/libdrupal.sh

```diff
@@ -280,6 +289,27 @@ drupal_site_install() {
     fi
 }
 
+drupal_create_config_directory() {
+	if [[ -z $DRUPAL_CONFIG_SYNC_DIR ]]; then
+		# In order to use Drush commands we need to have a value for
+		# config_sync_directory or it will throw an error so we add a
+		# temp value here first
+		drupal_conf_set "\$settings['config_sync_directory']" "temp" no
+		DRUPAL_CONFIG_SYNC_DIR="sites/default/files/config_$(drush eval "echo(Drupal\Component\Utility\Crypt::randomBytesBase64(55))")"
+		debug_execute mkdir "$DRUPAL_BASE_DIR/$DRUPAL_CONFIG_SYNC_DIR"
+	else
+		debug_execute mkdir "$@"
+	fi
+}
+
+drupal_set_hash_salt() {
+	if [[ -z $DRUPAL_HASH_SALT ]]; then
+		drupal_conf_set "\$settings['hash_salt']" "$(drush eval "echo(Drupal\Component\Utility\Crypt::randomBytesBase64(55))")" no
+	else
+		drupal_conf_set "\$settings['hash_salt']" "$DRUPAL_HASH_SALT" no
+	fi
```

Comment:
In the same way than before, this could be improved as follows:

```suggestion
    local hash_salt="${DRUPAL_HASH_SALT:-}"
    if is_empty_value "$hash_salt"; then
        hash_salt="$(generate_random_string -t alphanumeric -c 32)"
    fi
    drupal_conf_set "\$settings['hash_salt']" "$hash_salt"
```